documentation/sphinx/source/api-c.rst

```diff
@@ -642,6 +642,9 @@ Applications must provide error handling and an appropriate retry loop around th
 
     An enumeration of available opcodes to be passed to :func:`fdb_transaction_atomic_op()`
 
+    A information line for `atomic-add`, the result overflows according to the width
+    of param2.
+
     ``FDB_MUTATION_TYPE_ADD``
 
     |atomic-add1|
```

Comment:
|atomic-add1| and -add2 are substitutions, where the actual content comes from `.. |atomic-add1| replace::` in api-common.rst.inc.  I think it'd make more sense to expand the text there to be more specific about, that in the case of overflow, the result is truncated to the width of param2.